homeassistant/components/unmanic/entity.py

```diff
@@ -0,0 +1,32 @@
+"""Unmanic entity class."""
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN, NAME
+
+
+class UnmanicEntity(CoordinatorEntity):
+    """Class to represent an Unmanic entity."""
+
+    def __init__(self, coordinator, config_entry, device_id):
+        """Initialize."""
+        self.config_entry = config_entry
+        self._device_id = device_id
+        super().__init__(coordinator)
+
+    @property
+    def device_info(self):
```

Comment:
Since this is static, you can move this to `__init__` and set the attribute directly.

Example:
https://github.com/home-assistant/core/blob/d3374ecd8e0b07884713af77e539ebbf36d53881/homeassistant/components/overkiz/entity.py#L38